modules/sdk-test/src/bitgo/lib/config.ts

```diff
@@ -0,0 +1,201 @@
+import * as _ from 'lodash';
+import { Environments } from '@bitgo/sdk-core';
+import {
+  coins,
```

Comment:
If we are duplicating this from `bitgo` instead of removing/moving it, should these files be refactored into `statics` or `core`?

Reply:
@mmcshinsky-bitgo I just removed the config file from sdk-test as not needed anymore. It might be ended up in sdk-core like what mentioned.